src/tiler/server.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+var Windshaft = require('windshaft'),
+
+    workerCount = process.env.WORKERS || require('os').cpus().length,
+    port = process.env.PORT || 4000,
+
+    dbUser = process.env.NYC_TREES_DB_USER || 'nyc_trees',
+    dbPassword = process.env.NYC_TREES_DB_PASSWORD || 'nyc_trees',
+    dbHost = process.env.NYC_TREES_DB_HOST || 'localhost',
+    dbPort = process.env.NYC_TREES_DB_PORT || 5432,
+
+    redisHost = process.env.NYC_TREES_REDIS_HOST || 'localhost',
+    redisPort = process.env.NYC_TREES_REDIS_PORT || 6379,
+
+    config = {
+        base_url: '/:cache_buster/:dbname/:type',
+        base_url_notable: '/:cache_buster/:dbname',
+
+        grainstore: {
+            datasource: {
+                user: dbUser,
+                password: dbPassword,
+                host: dbHost,
+                port: dbPort,
+                geometry_field: 'geom',
+                srid: 4326
```

Comment:
I think so.  At the very least, it's as valid as hardcoding any other aspect of our DB schema, like table names, columns names, SQL statements, etc.